mewarpx/mewarpx/mwxrun.py

```diff
@@ -103,23 +104,50 @@ def init_timestep(self, V_anode, DT=None, CFL_factor=None, V_grid=5):
         return self.dt
 
 
-    def init_run(self):
+    def init_run(self, restart=None, checkpoint_dir="diags",
+                 checkpoint_prefix="chkpoint", additional_steps=None):
         if self.initialized:
             raise RuntimeError(
                 "Attempted to initialize the mwxrun class multiple times.")
-
-        self.simulation.initialize_inputs()
-        self.simulation.initialize_warpx()
-
-        self.me = _libwarpx.libwarpx.warpx_getMyProc()
-        self.n_procs = _libwarpx.libwarpx.warpx_getNProcs()
-
-        # A level is needed for many things like level number. For now I'm
-        # statically setting the default level here. I'm not sure of pitfalls
-        # or how to handle it more generally yet.
-        self.lev = _libwarpx.libwarpx.warpx_finestLevel()
-
-        self.initialized = True
+        try:
+            restart_success = None
+            if restart != False:
+                restart_success = init_restart_util.run_restart(
+                    checkpoint_dir, checkpoint_prefix,
+                    restart is not None, additional_steps
+                )
```

Comment:
Passing `restart is not None` to `run_restart` seems a bit weird, and doesn't immediately convey what is actually happening here. I think it'd be helpful and make the code easier to read if you created another variable called `force_restart` like this: 

```suggestion
        # force_restart is False by default unless restart is set to True
        force_restart = restart if isinstance(restart, bool) default False
        try:
            restart_success = None
            if restart != False:
                restart_success = init_restart_util.run_restart(
                    checkpoint_dir, checkpoint_prefix,
                    force_restart, additional_steps
                )
```

Reply:
After looking a little, it turns out `None` is falsy in python so I can just do `bool(restart)`

Reply:
I could do `force_restart = True if bool(restart) else False` but this doesn't feel necessary to me, because that's what `bool(restart)` means anyway. Let me know what you think.

Reply:
`bool(restart)` works. I think the advantage of making a second variable is just making it clear that you're passing whether or not to force a restart, no whether or not you should restart.